app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CollaboratorsController < ApplicationController
+  def index
+  end
+
+  def new
+    @collaborator = Collaborator.new
+    @user = User.all
```

Comment:
Not a huge error, and won't break the code, but it's helpful to people unfamiliar with the code (which will include you in two months) if the plurality of the variable matches what it contains. So arrays and hashes are typically pluralized, so in this case @user is an array of Users and should be plural.